uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -712,6 +712,22 @@ public function upgrade_1031() {
     return TRUE;
   }
 
+  /**
+   * Upgrade CustomGroup, setting Probation and Activity_Custom_Fields to is_reserved Yes
+   *
+   * @return bool
+   */
+  public function upgrade_1032() {
+    civicrm_api3('CustomGroup', 'get', [
+      'sequential' => 1,
+      'return' => ['id'],
+      'name' => ['IN' => ['Probation', 'Activity_Custom_Fields']],
```

Comment:
The PR description only mentions "Activity_Custom_Fields", which is correct?

Reply:
This PR only cater for 'Activity_Custom_Fields'. Other changes are in a separate PR. That is they are excluded. Same with the other PR, Activity_Custom_Fields was excluded.